runtime/doc/treesitter.txt

```diff
@@ -448,56 +467,59 @@ get_node_at_position({bufnr}, {row}, {col}, {opts})
                    (default true)
 
     Return: ~
-        (table) Named node under the cursor
+        userdata |tsnode| under the cursor
 
 get_node_range({node_or_range})                             *get_node_range()*
     Get the node's range or unpack a range table
 
     Parameters: ~
-        {node_or_range}  (table)
+        {node_or_range}  (userdata|table) |tsnode| or table of positions
 
     Return: ~
-        (table) start_row, start_col, end_row, end_col
+        (table) `{ start_row, start_col, end_row, end_col }`
 
 get_parser({bufnr}, {lang}, {opts})                             *get_parser()*
-    Gets the parser for this bufnr / ft combination.
+    Gets the parser for a specific buffer and filetype.
 
     If needed this will create the parser. Unconditionally attach the provided
     callback
 
     Parameters: ~
         {bufnr}  (number|nil) Buffer the parser should be tied to (default:
                  current buffer)
-        {lang}   (string) |nil Filetype of this parser (default: buffer
+        {lang}   (string|nil) Filetype of this parser (default: buffer
                  filetype)
         {opts}   (table|nil) Options to pass to the created language tree
 
     Return: ~
-        (table) Parser object
+        LanguageTree |LanguageTree| object to use for parsing
 
 get_string_parser({str}, {lang}, {opts})                 *get_string_parser()*
     Gets a string parser
 
     Parameters: ~
-        {str}   The string to parse
-        {lang}  The language of this string
-        {opts}  Options to pass to the created language tree
+        {str}   (string) Text to parse
+        {lang}  (string) Language of this string
+        {opts}  (table|nil) Options to pass to the created language tree
+
+    Return: ~
+        LanguageTree |LanguageTree| object to use for parsing
 
 is_ancestor({dest}, {source})                                  *is_ancestor()*
     Determines whether a node is the ancestor of another
 
     Parameters: ~
-        {dest}    (table) Possible ancestor
-        {source}  (table) Possible descendant node
+        {dest}    userdata Possible ancestor |tsnode|
+        {source}  userdata Possible descendant |tsnode|
 
     Return: ~
-        (boolean) True if dest is an ancestor of source
+        (boolean) True if `dest` is an ancestor of `source`
```

Comment:
Huh that's a little disappointing, I'd expect that the `@p` annotations would be printted as `{dest}`. We'll have to fix that.

Reply:
That's just how Doxygen formats these, I'm afraid. It might be easier to format them manually again.

Reply:
Yeah, let's not use those -- lua-language-server doesn't interpret them, either.